src/main/java/com/camacho/simpleannotation/exceptions/BadAnnotationUsageException.java

```diff
@@ -1,4 +1,4 @@
-package com.example.simpleannotation.exceptions;
+package com.camacho.simpleannotation.exceptions;
 
 public class BadAnnotationUsageException extends Exception{
```

Comment:
add space betweet Exception and {

Reply:
Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)